src/Spice86.Core/Emulator/Devices/Input/Keyboard/Keyboard.cs

```diff
@@ -56,12 +56,12 @@ public class Keyboard : DefaultIOPortHandler {
             if (lastKeyboardInput.IsPressed) {
                 scancode = _keyScanCodeConverter?.GetKeyPressedScancode(lastKeyboardInput);
                 if (_loggerService.IsEnabled(Serilog.Events.LogEventLevel.Information)) {
-                    _loggerService.Information("Getting scancode. Key pressed {@KeyCode} scancode {@ScanCode}", LastKeyboardInput.Value.EventArgs, scancode);
+                    // _loggerService.Information("Getting scancode. Key pressed {@KeyCode} scancode {@ScanCode}", LastKeyboardInput.Value.EventArgs, scancode);
```

Comment:
Re-enable code ?

Reply:
Shoot, that wasn't meant to be part of the commit.

Reply:
Restored